designer/server/src/test/scala/pl/touk/nussknacker/ui/api/testing/JsonSchemalessAdHocTestsSpec.scala

```diff
@@ -90,6 +96,10 @@ class JsonSchemalessAdHocTestsSpec
       "scenarioTypes.streaming.modelConfig.components.kafka.config.kafkaProperties",
       fromMap(Map("bootstrap.servers" -> "localhost:8070", "schema.registry.url" -> "http://localhost:8069").asJava)
     )
+    .withValue(
+      "scenarioTypes.streaming.modelConfig.components.kafka.config.useDataSampleParamForSchemalessJsonTopicBasedKafkaSource",
```

Comment:
Is that config value described anywhere? It seems that it is the only usage. Is it needed?

Reply:
Data sample params for Kafka Sources are disabled by default. This is an internal feature for now and applies only to a specific kind of Kafka Sources (UniversalKafkaSource without schema, with JSON content type). That's why it is not mentioned in the docs. We need to enable this feature to test the default value for ad-hoc tests.